<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureGeneral</class>
 <widget class="QWidget" name="ConfigureGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="general_group">
       <property name="title">
        <string>General</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QCheckBox" name="toggle_check_exit">
          <property name="text">
           <string>Confirm exit while emulation is running</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="toggle_background_pause">
          <property name="text">
           <string>Pause emulation when in background</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="toggle_background_mute">
          <property name="text">
           <string>Mute audio when in background</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="toggle_hide_mouse">
          <property name="text">
           <string>Hide mouse on inactivity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="toggle_gamemode">
          <property name="text">
           <string>Enable Gamemode</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="updateBox">
       <property name="title">
        <string>Updates</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="toggle_update_check">
          <property name="text">
           <string>Check for updates on start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="toggle_auto_update">
          <property name="text">
           <string>Silently auto update after closing</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="emulation_group">
       <property name="title">
        <string>Emulation</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget_region" native="true">
          <layout class="QHBoxLayout" name="region_layout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="region_label">
             <property name="text">
              <string>Region:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="region_combobox">
             <item>
              <property name="text">
               <string>Auto-select</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">JPN</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">USA</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">EUR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">AUS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">CHN</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">KOR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">TWN</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="emulation_speed_layout" native="true">
          <layout class="QHBoxLayout" name="emulation_speed_layout_inner">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QComboBox" name="emulation_speed_combo">
             <item>
              <property name="text">
               <string>Use global emulation speed</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Set emulation speed:</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_emulation_speed">
             <property name="text">
              <string>Emulation Speed:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="frame_limit">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>199</number>
             </property>
             <property name="singleStep">
              <number>5</number>
             </property>
             <property name="pageStep">
              <number>15</number>
             </property>
             <property name="value">
              <number>19</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="emulation_speed_display_label">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="screenshot_group">
       <property name="title">
        <string>Screenshots</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <widget class="QWidget" name="widget_screenshot" native="true">
          <layout class="QHBoxLayout" name="screenshot_layout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QComboBox" name="screenshot_combo">
             <item>
              <property name="text">
               <string>Use global screenshot path</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Set screenshot path:</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="screenshot_dir_label">
             <property name="text">
              <string>Save Screenshots To</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="screenshot_dir_path"/>
           </item>
           <item>
            <widget class="QToolButton" name="change_screenshot_dir">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="button_reset_defaults">
       <property name="text">
        <string>Reset All Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>toggle_check_exit</tabstop>
  <tabstop>toggle_background_pause</tabstop>
  <tabstop>toggle_hide_mouse</tabstop>
  <tabstop>toggle_update_check</tabstop>
  <tabstop>toggle_auto_update</tabstop>
  <tabstop>button_reset_defaults</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
